testutil/network/network.go

```diff
@@ -0,0 +1,138 @@
+package network
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spm/cosmoscmd"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+func New(t *testing.T, config network.Config, genAccNames ...string) *network.Network {
+	kr := generateKeyring(t)
+
+	// add genesis accounts
+	genAuthAccs := make([]authtypes.GenesisAccount, len(genAccNames))
+	genBalances := make([]banktypes.Balance, len(genAccNames))
+	mnemonics := make([]string, len(genAccNames))
+	for i, name := range genAccNames {
+		a, b, mnm := newGenAccout(kr, name, 1000000000000)
+		genAuthAccs[i] = a
+		genBalances[i] = b
+		mnemonics[i] = mnm
+	}
+
+	config, err := addGenAccounts(config, genAuthAccs, genBalances)
+	if err != nil {
+		panic(err)
+	}
+
+	net := network.New(t, config)
+
+	// add the keys to the keyring that is used by the integration test
+	for i, name := range genAccNames {
+		_, err := net.Validators[0].ClientCtx.Keyring.NewAccount(name, mnemonics[i], "", "", hd.Secp256k1)
+		require.NoError(t, err)
+	}
+
+	// t.Cleanup(net.Cleanup)
```

Comment:
What is this line for?

Reply:
it was adding a cleanup function to a test, but it ended being redundant. forgot to remove it, good catch!

https://github.com/celestiaorg/celestia-app/pull/171/commits/84baa00969501d419eb30e3e6410cb553a63fc06